packages/demo/examples/TSDigisolPlatform/configs/layouts/dashboardLayout.js

```diff
@@ -43,8 +43,8 @@ import { EditOrderLineAddressDetailComponent } from "../../components/domainSpec
 // ******************** TEST OF AddEditEntity *************************
 import { AddEditEntity } from "../../../../components/AddEditEntity";
 
-// ******************** POC on NAV BAR *****************************
-import { ToggleNavigation } from "../../../../components/ToggleNavigation";
+// ******************** POC on NEWNAV BAR *****************************
+import { NewNavbar } from "../../components/NewNavbar/index";
```

Comment:
Hi @rajatkhade 

Can you let me know where does you have appConfig 
for our repo We are using appConfig written in layout.js file 
Which file you are using ?

Reply:
I'm also using the same